geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/RebalanceCommandDistributedTest.java

```diff
@@ -14,143 +14,76 @@
+  public static GfshCommandRule gfsh = new GfshCommandRule();
+
+  private static MemberVM locator, server1, server2;
 
   @Before
   public void before() throws Exception {
-    setUpJmxManagerOnVm0ThenConnect(null);
-    setupTestRebalanceForEntireDS();
+    locator = cluster.startLocatorVM(0);
+
+    int locatorPort = locator.getPort();
+    server1 = cluster.startServerVM(1, "localhost", locatorPort);
+    server2 = cluster.startServerVM(2, "localhost", locatorPort);
+
+    MemberVM.invokeInEveryMember(() -> setUpRegion(), server1, server2);
+
+    gfsh.connectAndVerify(locator);
   }
 
   @Test
   public void testSimulateForEntireDSWithTimeout() {
     // check if DistributedRegionMXBean is available so that command will not fail
-    final VM manager = Host.getHost(0).getVM(0);
-    manager.invoke(() -> checkRegionMBeans());
-
-    getLogWriter().info("testSimulateForEntireDS verified MBean and executing command");
+    locator.waitTillRegionsAreReadyOnServers("/" + REBALANCE_REGION_NAME1, 2);
+    locator.waitTillRegionsAreReadyOnServers("/" + REBALANCE_REGION_NAME2, 2);
 
     String command = "rebalance --simulate=true --time-out=-1";
 
-    CommandResult cmdResult = executeCommand(command);
-
-    getLogWriter().info("testSimulateForEntireDS just after executing " + cmdResult);
-
-    assertThat(cmdResult).isNotNull();
-
-    String stringResult = commandResultToString(cmdResult);
-    getLogWriter().info("testSimulateForEntireDS stringResult : " + stringResult);
-    assertThat(cmdResult.getStatus()).isEqualTo(Result.Status.OK);
+    gfsh.executeAndAssertThat(command).statusIsSuccess();
   }
 
-  private void checkRegionMBeans() {
-    WaitCriterion waitForManagerMBean = new WaitCriterion() {
-      @Override
-      public boolean done() {
-        final ManagementService service = ManagementService.getManagementService(getCache());
-        final DistributedRegionMXBean bean =
-            service.getDistributedRegionMXBean(Region.SEPARATOR + REBALANCE_REGION_NAME);
-        if (bean == null) {
-          getLogWriter().info("Still probing for checkRegionMBeans ManagerMBean");
-          return false;
-        } else {
-          // verify that bean is proper before executing tests
-          return bean.getMembers() != null && bean.getMembers().length > 1
-              && bean.getMemberCount() > 0
-              && service.getDistributedSystemMXBean().listRegions().length >= 2;
-        }
-      }
-
-      @Override
-      public String description() {
-        return "Probing for testRebalanceCommandForSimulateWithNoMember ManagerMBean";
-      }
-    };
-
-    waitForCriterion(waitForManagerMBean, 2 * 60 * 1000, 2000, true);
-
-    DistributedRegionMXBean bean = ManagementService.getManagementService(getCache())
-        .getDistributedRegionMXBean("/" + REBALANCE_REGION_NAME);
-    assertThat(bean).isNotNull();
-  }
+  private static void setUpRegion() {
```

Comment:
I don't fully understand why the original test was setting up the regions that way, but your way is different from what original test is doing. The original test is putting in different values in each VM to the same region, and also putting values in other regions (different for each VM) as well. It would be best to understand if this has something to do with what the test is trying to test, or if not, at least do the same setup as the original test was doing.